lms/models/application_instance.py

```diff
@@ -61,6 +62,25 @@ def decrypted_developer_secret(self, aes_secret):
 
         return cipher.decrypt(self.developer_secret)
 
+    def lms_host(self):
+        """
+        Return the hostname part of this ApplicationInstance's lms_url.
+
+        For example if application_instance.lms_url is
+        "https://example.com/lms/" then application_instance.lms_host() will
+        return "example.com".
+
+        :raise ValueError: if the ApplicationInstance's lms_url can't be parsed
+        """
+        # urlparse() or .netloc will raise ValueError for some invalid URLs.
+        lms_host = urlparse(self.lms_url).netloc
+
+        # For some URLs urlparse(url).netloc returns an empty string.
+        if not lms_host:
+            raise ValueError()
```

Comment:
```suggestion
            raise ValueError("No net location extracted from url")
```
... or good words... you know.

Reply:
Added a string